.github/workflows/release.yml

```diff
@@ -0,0 +1,27 @@
+name: release
+
+on:
+  push:
+    tags:
+      - "v[0-9]+.[0-9]+.[0-9]+"
```

Comment:
are we still planning to support tags of the form `v0.11.rc0`?

Reply:
we could consider switching to `v` as it is the default on github

v0.0.0
or 0.0.0

Reply:
I meant the `rc0` at the end. The regex only supports digits.

Reply:
currently we don't have it in on-tag. we could add it. but i think rc could be an expection where we do it manually in the GUI